quarkchain/core.py

```diff
@@ -1115,6 +1115,7 @@ class CrossShardTransactionDeposit(Serializable):
         ("gas_token_id", uint64),
         ("transfer_token_id", uint64),
         ("gas_remained", uint256),
+        ("refund_rate", uint256),
```

Comment:
I think we don't need it if we can just convert the gas price

Reply:
a second thought: converting gas price doesn't seem enough because we need the refund rate to determine how much should be sent to the miner and how much should be burned

cc @qizhou